lib/caseStatus.ts

```diff
@@ -15,3 +15,13 @@ export const getCaseStatusByPersonId = async (
   );
   return data;
 };
+
+export const GetFormValues = async (type: string): Promise<FormFields> => {
+  const { data }: { data: FormFields } = await axios.get(
+    `${ENDPOINT_API}/case-status/form-options/${type}`,
```

Comment:
I think this is fine but I'm not sure if it fully matches the backend API?

```
[ProducesResponseType(typeof(GetCaseStatusFieldsResponse), StatusCodes.Status200OK)]
        [HttpGet]
        [Route("{type?}")]
        public IActionResult GetCaseStatusTypeFields([FromRoute] GetCaseStatusFieldsRequest request)
        {
            try
            {
                return Ok(_getCaseStatusFieldsUseCase.Execute(request));
            }
            catch (CaseStatusNotFoundException exception)
            {
                return NotFound(exception.Message);
            }
        }
```

I think it should just be 

```
[Route("{type}")]
        public IActionResult GetCaseStatusTypeFields(string type)
```

Maybe this API call is working fine because .net is managing to figure out what was intended if if the code is a bit odd.

Reply:
Hi John, I'll have a look at the backend code and do the necessary edits then!

I haven't made that endpoint so I can't really say why, but I'll make those edits and checks if everything works :)